lib/components/SButton.vue

```diff
@@ -705,6 +705,7 @@ function handleClick(): void {
 
 .SButton.block {
   display: flex;
+  justify-content: center;
```

Comment:
[SSheet had this misaligned.](https://story.sefirot.globalbrains.com/story/stories-components-ssheet-01-playground-story-vue?variantId=_default) So fixed it too.

Reply:
Thanks! 🙌